infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/ValidatorLogger.java

```diff
@@ -73,11 +74,19 @@ public void dutyFailed(
       final UInt64 slot,
       final Set<String> maybeKey,
       final Throwable error) {
-    final String errorString =
-        String.format(
-            "%sFailed to produce %s  Slot: %s%s",
-            PREFIX, producedType, slot, formatValidators(maybeKey));
-    log.error(ColorConsolePrinter.print(errorString, Color.RED), error);
+    if (!Throwables.getRootCause(error).getMessage().contains("signer is no longer active")) {
+      final String errorString =
+          String.format(
+              "%sFailed to produce %s  Slot: %s%s",
+              PREFIX, producedType, slot, formatValidators(maybeKey));
+      log.error(ColorConsolePrinter.print(errorString, Color.RED), error);
+    } else {
+      final String errorString =
+          String.format(
+              "%sValidator deleted, cannot produce %s  Slot: %s%s",
+              PREFIX, producedType, slot, formatValidators(maybeKey));
+      log.info(ColorConsolePrinter.print(errorString, Color.YELLOW));
```

Comment:
wasn't sure what the best answer was here... i think I'm ok with info, but not sure about the colour...

Reply:
I think I'd leave it yellow but changing the text to:
`%sValidator deleted, skipping previously scheduled %s production. Slot: %s%s`